tools/swift-localize-diagnostics/swift-localize-diagnostics.cpp

```diff
@@ -0,0 +1,105 @@
+//===--- swift-localize-diagnostics.cpp -----------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// Create a YAML file from the diagnostic messages texts in `.def` files.
```

Comment:
This is not a localization tool it's a converter from one format into the other - let's rename it to something like `swift-def-to-yaml-converter`.